Traceback (most recent call last):
  File "train.py", line 439, in <module>
    train(data_dir, model_dir, args)
  File "train.py", line 236, in train
    mask_out, gender_out, age_out = model(inputs)
  File "/opt/conda/lib/python3.8/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/opt/ml/code/ImageClassification/model.py", line 73, in forward
    x = self.feature_net(x)
  File "/opt/conda/lib/python3.8/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/opt/conda/lib/python3.8/site-packages/efficientnet_pytorch/model.py", line 314, in forward
    x = self.extract_features(inputs)
  File "/opt/conda/lib/python3.8/site-packages/efficientnet_pytorch/model.py", line 296, in extract_features
    x = block(x, drop_connect_rate=drop_connect_rate)
  File "/opt/conda/lib/python3.8/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/opt/conda/lib/python3.8/site-packages/efficientnet_pytorch/model.py", line 107, in forward
    x = self._swish(x)
  File "/opt/conda/lib/python3.8/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/opt/conda/lib/python3.8/site-packages/efficientnet_pytorch/utils.py", line 80, in forward
    return SwishImplementation.apply(x)
  File "/opt/conda/lib/python3.8/site-packages/efficientnet_pytorch/utils.py", line 67, in forward
    result = i * torch.sigmoid(i)
RuntimeError: CUDA out of memory. Tried to allocate 1.69 GiB (GPU 0; 31.75 GiB total capacity; 3.93 GiB already allocated; 1.68 GiB free; 5.89 GiB reserved in total by PyTorch)